knimin/lib/ag_jira.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+from jira import JIRA
+
+
+def jira_connect(server='https://jira.atlassian.com',
+                 user=None, password=None):
+    """Connect to JIRA server
+
+    Parameters
+    ----------
+    server: str, optional
```

Comment:
why is server optional? Can it work without specifying a server? I would vote for making server mandatory